src/domain/Effort.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package domain;
+
+/**
+ *
+ * @author Lasse
+ */
+public class Effort {
+
+  private int TotalPrice;
```

Comment:
All attributes should be camelcased, `TotalPrice` should be `totalPrice` instead